content/docs/pulumi-cloud/projects-and-stacks.md

```diff
@@ -102,11 +102,75 @@ Examples for adding the Stack Output `readme` to a Pulumi program:
 {{% choosable language typescript %}}
 
 ```typescript
+import * as pulumi from "@pulumi/pulumi";
+import * as aws from "@pulumi/aws";
 import { readFileSync } from "fs";
-export const strVar = "foo";
-export const arrVar = ["fizz", "buzz"];
-// add readme to stack outputs. must be named "readme".
-export const readme = readFileSync("./Pulumi.README.md").toString();
+
+const loggroup = new aws.cloudwatch.LogGroup("loggroup", {});
+const dashboard = new aws.cloudwatch.Dashboard("dashboard", {
+    dashboardName: "mydashboard",
+    dashboardBody: JSON.stringify({
+        widgets: [
+            {
+                type: "metric",
+                x: 0,
+                y: 0,
+                width: 12,
+                height: 6,
+                properties: {
+                    metrics: [[
+                        "AWS/EC2",
+                        "CPUUtilization",
+                        "InstanceId",
+                        "i-012345",
+                    ]],
+                    period: 300,
+                    stat: "Average",
+                    region: "us-east-1",
+                    title: "EC2 Instance CPU",
+                },
+            },
+            {
+                type: "text",
+                x: 0,
+                y: 7,
+                width: 3,
+                height: 3,
+                properties: {
+                    markdown: "Hello world",
+                },
+            },
+        ],
+    }),
+});
+const database = new aws.rds.Cluster("database", {
+    clusterIdentifier: "aurora-cluster-demo",
+    engine: aws.rds.EngineType.AuroraPostgresql,
+    availabilityZones: [
+        "us-west-2a",
+        "us-west-2b",
+        "us-west-2c",
+    ],
+    databaseName: "mydb",
+    masterUsername: "foo",
+    masterPassword: "must_be_eight_characters",
+    backupRetentionPeriod: 5,
+    preferredBackupWindow: "07:00-09:00",
+    skipFinalSnapshot: true,
+});
+const instance = new aws.rds.ClusterInstance("instance", {
+    clusterIdentifier: database.clusterIdentifier,
+    instanceClass: aws.rds.InstanceType.R5_Large,
+    engine: "aurora-postgresql",
+    engineVersion: "15.4",
+    performanceInsightsEnabled: true,
+});
+
+export const dashboardName = dashboard.dashboardName;
+export const cloudwatchLogGroup = loggroup.name;
+export const instanceId = instance.id;
+export const dbResourceId = instance.dbiResourceId;
+export const readme = readFileSync("./Pulumi.readme.md").toString();
```

Comment:
Isn't this really the _only_ useful line for purposes of illustrating this concept  (separate, perhaps, from an additional `Pulumi.readme.md` go along with it)? I'm not sure I understand what purpose the rest of the code serves, given the `readme` export doesn't seem to reference any of it. (Sorry if I'm completely misunderstanding what you're aiming for here, though.)

Reply:
The previous version didn't actually work. This you can copy and paste